netdata.spec.in

```diff
@@ -42,9 +54,13 @@ Requires(pre): /usr/sbin/groupadd
 Requires(pre): /usr/sbin/useradd
 
 %if %{with systemd}
+%if 0%{?suse_version}
+%{?systemd_requires}
+%else
```

Comment:
not critical, these can also happen in single conditional at prologue as far as I understand, have you tried it?
```
%xxx() %{expand:\
xxx
yyy
}
```
see examples at /etc/rpm/macros.perl
http://backreference.org/2011/09/17/some-tips-on-rpm-conditional-macros/